### DISCUSSION TITLE
Unable to open FModel.exe

### DISCUSSION THREAD

**Discussion by Basdower** [4sval/FModel#528] [Q&A]
Every time I try to open it, I keep getting the "_An unhandled exception occurred: FModel cannot be run from an archive file. Please extract it and try again._" error. Already have .net installed

**Comment by mccantym-personal** [comment]
> Every time I try to open it, I keep getting the "_An unhandled exception occurred: FModel cannot be run from an archive file. Please extract it and try again._" error. Already have .net installed

This is marked closed and unanswered; did you find a solution on your own, and if so, what was it?  If not, I'll have to reopen this with my own experience.  FModel is extracted to its own folder and .NET SDK 9.0.3 is installed, but I still get this warning, whether running normally or as Administrator.

EDIT: It looks like @GMatrixGames closed this the same day it was opened, is there an existing answer elsewhere?  I couldn't find one.

SECOND EDIT: I have finally figured it out after examining the source code that generates the Exception.  To any Windows 11 users who find this, ensure that the directory you've extracted FModel to is not within Win11's terrible OneDrive parent directory.
